scripts/src/main/resources/scripts/command/vscode

```diff
@@ -94,13 +97,12 @@ function doConfigureVsCode() {
   if [ -n "${mode}" ]
   then
     doConfigureWorkspace "${SETTINGS_PATH}/vscode/workspace" "${WORKSPACE_PATH}" "${mode}"
+    doConfigureWorkspace "${SETTINGS_PATH}/vscode/conf/vscode" "${VSCODE_CONF}" "${mode}"
```

Comment:
This does not make any sense to me. There is no such folder:
https://github.com/devonfw/ide-settings/tree/master/vscode
IMHO this is all a missunderstanding of the way devonfw-ide works and is designed.
Can you provide more details about your use-case and what you actually want to archive?

However, there is this folder that acts as a template for configuration during initial setup:
https://github.com/devonfw/ide-settings/tree/master/devon/

If you just want to have some initial config that preconfigures something for the end-user, just put it inside this structure. This is documented here:
https://github.com/devonfw/ide/blob/master/documentation/settings.asciidoc

Reply:
Maybe looking at the other PR i have will clear things up a little: https://github.com/devonfw/ide-settings/pull/15/files

So for background: VS Code has some settings with unacceptable defaults (telemetry, but there will probably be more...). So we need to configure those settings.

It is probably not acceptable to require our developers to re-install devon completely when we notice a missing setting, so using the template for configuration during initial setup is probably out of the question.

The settings this concerns reside exactly in the folder "${DEVON_IDE_HOME}/conf/vscode", so this PR simply makes that folder managed like the other configuration folder.